### ISSUE TITLE
Adblocker message broke the options

### ISSUE THREAD

**Issue by yraropmet** [photopea/photopea#4649]
I was working on a project, when I was trying to export it the pop-up message indicating that I have an adblocker appeared and I clicked below it (I missed it) and it disappeared. Since that moment, I was never able to download anything through the menu. I could only download using the buttons "PNG" and "SVG" on the top of the screen. It broke everything (and tho ads are still showing even with my adblocker). Nice feature, nice feature.

**Comment by photopea** [comment]
Could you please describe how can I reproduce the problem?

**Comment by yraropmet** [comment]
I really have no idea. I just drew something, clicked the "File" button, hover the "Export As..." option, then selected "SVG". I checked the "add hidden layers" box (It loads very slowly, but I had many layers (around 20)) then clicked on "Save". The first adblocker message appeared, I clicked it, then goes the second one. I missed it and clicked below the button (I don't quite remember but I think I clicked directly on the "void", where there was no button at all and just the zone where there was what I had drawn) and the message disappeared, but the third and last pop-up message never appeared. Then, whatever I did, each and every way to export a project (As PNG, JPG, SVG...) stopped working. Their "Save" button, which, once clicked pop in the adblocker messages, was functional and interactive, but no longer exported a thing. The only way I had to get my project in SVG or PNG was selecting every layer of it and click the button on the top of the screen (which don't pop any adblocker message window).
